File: android/Reader/app/src/main/java/com/enrimilan/reader/model/SensorData.java
package com.enrimilan.reader.model;

public class SensorData {

    private double temperature;
    private int humidity;
    private int battery;

    public double getTemperature() {
        return temperature;
    }

    public void setTemperature(double temperature) {
        this.temperature = temperature;
    }

    public int getHumidity() {
        return humidity;
    }

    public void setHumidity(int humidity) {
        this.humidity = humidity;
    }

    public int getBattery() {
        return battery;
    }

    public void setBattery(int battery) {
        this.battery = battery;
    }

}

File: javascript/src/main/model/sensor-data.js
"use strict";
class SensorData {

	getTemperature() {
        return this.temperature;
    }

    setTemperature(temperature) {
        this.temperature = temperature;
    }

    getHumidity() {
        return this.humidity;
    }

    setHumidity(humidity) {
        this.humidity = humidity;
    }

    getBattery() {
        return this.battery;
    }

    setBattery(battery) {
        this.battery = battery;
    }
}

module.exports = SensorData;